liblangutil/SourceReferenceFormatter.h

```diff
@@ -51,22 +51,23 @@ class SourceReferenceFormatter
 
 	/// Prints source location if it is given.
 	void printSourceLocation(SourceReference const& _ref);
-	void printExceptionInformation(SourceReferenceExtractor::Message const& _msg);
-	void printExceptionInformation(util::Exception const& _exception, std::string const& _severity);
+	void printExceptionInformation(SourceReferenceExtractor::Message const& _msg, bool _printFullType=false);
+	void printExceptionInformation(util::Exception const& _exception, Error::Type _type, bool _printFullType=false);
 	void printErrorInformation(langutil::ErrorList const& _errors);
 	void printErrorInformation(Error const& _error);
 
 	static std::string formatExceptionInformation(
 		util::Exception const& _exception,
-		std::string const& _name,
+		Error::Type _type,
 		CharStreamProvider const& _charStreamProvider,
 		bool _colored = false,
-		bool _withErrorIds = false
+		bool _withErrorIds = false,
+		bool jsonOutput = false
```

Comment:
```suggestion
		bool _jsonOutput = false
```

Reply:
Good catch, this also should have been renamed as the other instances of this parameter.
```suggestion
		bool _printFullType = false
```